internal/collector/instance.go

```diff
@@ -144,22 +147,37 @@ func AddToPod(
 
 // startCommand generates the command script used by the collector container
 func startCommand(includeLogrotate bool) []string {
-	var startScript = `
-/otelcol-contrib --config /etc/otel-collector/config.yaml
-`
-
+	var logrotateCommand string
 	if includeLogrotate {
-		startScript = `
-/otelcol-contrib --config /etc/otel-collector/config.yaml &
+		logrotateCommand = `logrotate -s /tmp/logrotate.status /etc/logrotate.d/logrotate.conf`
+	}
+
+	var startScript = fmt.Sprintf(`
+OTEL_PIDFILE=/tmp/otel.pid
```

Comment:
Does the `otelcontrib` process have a name? Or did we want the PID to check if it's running (without recourse to `pgrep`)?

Reply:
I'm honestly not sure of the best way to do this. I used some prior art in our codebase as a starting point